src/data-access/MwApiLangCodeRetriever.ts

```diff
@@ -5,17 +5,22 @@ import { processWbGetClaimsResponse, WbGetClaimsResponse } from './apiBasedLangC
 export default class MwApiLangCodeRetriever implements LangCodeRetriever {
 
 	private readonly api: MwApi;
-	private readonly languageCodeProperty: string;
+	private readonly languageCodeProperty: string | null;
 
-	public constructor( api: MwApi, languageCodeProperty: string ) {
-		if ( typeof languageCodeProperty !== 'string' ) {
-			throw new Error( `Expected languageCodeProperty to be propertyId but received ${languageCodeProperty} instead!` );
+	public constructor( api: MwApi, languageCodeProperty: string | null ) {
+		if ( !languageCodeProperty ) {
+			// eslint-disable-next-line no-console
+			console.warn( 'LexemeLanguageCodePropertyId seems not configured, language code inference will not work.' );
```

Comment:
I don’t think we should warn in this case, it’s not like end users of the wiki can do much about it unless they also happen to be sysadmins.

Reply:
mh, to me this feels like a deprecation warning, but since we can't enable it conditionally, we should probably remove it indeed.